### ISSUE TITLE
Installation failed 

### ISSUE THREAD

**Issue by ghost** [beercan1989/docker-arm-mysql#3]
Hi,
I am trying to install your package, but got the error code:
`Step 9/16 : RUN /opt/docker-arm-mysql/configure-mysql-for-docker.bash
 ---> Running in da2b8b803f04
OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"/bin/sh\": stat /bin/sh: no such file or directory": unknown`

Docker -v:
`Docker version 18.03.1-ce, build 9ee9f40`

`Distributor ID: Debian
Description:    Debian GNU/Linux 8.10 (jessie)
Release:        8.10
Codename:       jessie`

Thanks.

**Comment by beercan1989** [comment]
Hmmm strange, very hard to find any helpful information when Googling for that problem.

I've rebuilt from scratch with fresh images for `arm32v7/ubuntu:xenial` but under my `Raspbian GNU/Linux 9.4 (stretch)` I'm currently unable to reproduce it.

I've created a branch with some tweaks to the Dockerfile on `investigation-for-issue-3` but all that does is make sure all the files are bash and has less layers in the Dockerfile.

When checking out the codebase, is that in the Linux world or on Windows? A possible cause could be a line endings problem.

Then reading old issues such as https://github.com/moby/moby/issues/5135 makes it sound like it could be an issue with custom kernels or old ones that have a restricted layer support (but I'm not sure if that was referencing the moby vm or the host kernel).
